plugins/device/cmd/nvidia/nvml_client/client.go

```diff
@@ -0,0 +1,194 @@
+package nvml
```

Comment:
The containing folder should match the package name. So change nvml_client to nvml